src/controllers/auth.controller.ts

```diff
@@ -11,15 +11,19 @@ import { getRepository } from 'typeorm';
 import * as _ from 'lodash';
 import { User } from '@entities/user.entity';
 import { createJWT } from '@services/jwt.service';
+import { Mailer } from '@services/mailer.service';
 
 @JsonController('/auth')
 export class AuthController {
   @Post('/signup')
   async signUp(@Body({ validate: false }) user: User, @Res() response: any) {
-    let newUser;
+    let newUser: User;
 
     try {
       newUser = await getRepository(User).save(user);
+      new Mailer().sendMail(newUser.email, 'Welcome', Mailer.WELCOME, {
```

Comment:
just an idea, we could maybe inject these kinds of services as part of the controller-constructor, to avoid call the service constructor each time we use it and directly make `this.mailService.send()` or something like that.